packages/shorebird_cli/lib/src/commands/upgrade_command.dart

```diff
@@ -56,6 +56,8 @@ class UpgradeCommand extends ShorebirdCommand with ShorebirdVersionMixin {
       return ExitCode.success.code;
     }
 
+    logger.info('Updating Shorebird to $latestVersion from $currentVersion...');
```

Comment:
Thoughts on logging the new version only if the upgrade was successful? E.g “Upgraded to ${newVersion}”

Reply:
@felangel Can update to that too, but I created like this because while `Flutter Upgrade`, we get same message like <i>Upgrading Flutter from vX to xY<i>

Reply:
Good call seems reasonable to me to keep it consistent 👍

Reply:
Can we add a test for this as well?